### PR TITLE
[AMORO-1877]Support display paimon Operations in 'operations' page 

### LINKED ISSUE #1877: [Subtask]: Support display paimon Operations in 'operations' page
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/arctic/issues?q=is%3Aissue) and found no similar issues.

### Description

_No response_

### Parent issue

#1269 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by shidayang**
## Why are the changes needed?
Close #1877 .

## Brief change log
Add DDLReverser class to reverse the ddl sql for iceberg and paimon

## How was this patch tested?

- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / **no**)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/IcebergTableMetaExtract.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.component.reverser;
+
+import com.google.common.collect.Lists;
```
Comment (baiyangtx) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/IcebergTableMetaExtract.java:21]: Use shaded guave.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/IcebergTypeToSparkType.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import org.apache.spark.sql.types.MapType$;
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StringType$;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType$;
+import org.apache.spark.sql.types.TimestampType$;
+
+import java.util.List;
+
+public class IcebergTypeToSparkType extends TypeUtil.SchemaVisitor<DataType> {
```
Comment (baiyangtx) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/IcebergTypeToSparkType.java:46]: Add a statement indicating where the source code comes from.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/PaimonTypeToSparkType.java [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.types.UserDefinedType;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Utils for spark {@link DataType}.
+ */
```
Comment (baiyangtx) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/PaimonTypeToSparkType.java:55]: Add a statement indicating where the source code comes from.

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/TableMetaExtract.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.component.reverser;
+
+import com.google.common.base.Objects;
```
Comment (baiyangtx) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/component/reverser/TableMetaExtract.java:21]: Use shaded guava

**Review by baiyangtx — APPROVED** [review]
LGTM, thanks for your contribution.
